lighthouse-core/audits/seo/manual/mobile-friendly.js

```diff
@@ -6,6 +6,16 @@
 'use strict';
 
 const ManualAudit = require('../../manual/manual-audit');
+const i18n = require('../../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Description of a Lighthouse audit that provides detail on how well the page is setup to be compatible with mobile devices such as cellphones. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Take the [Mobile-Friendly Test](https://search.google.com/test/mobile-friendly) to check for audits not covered by Lighthouse, like sizing tap targets appropriately. [Learn more](https://developers.google.com/search/mobile-sites/).',
+  /** Title of a Lighthouse audit that provides detail on how well the page is setup to be compatible with mobile devices such as cellphones. This is displayed in a list of audit titles that Lighthouse generates. */
```

Comment:
I think we should tweak these manual ones a bit, it doesn't really provide detail on how well the page is setup, it tells the user to go do something else but the title doesn't really let any of that factor in :/

maybe something more like `Title of a Lighthouse audit that prompts users to manually check their page for compatibility with mobile devices` or something?